source/dialog/supports_init.hpp

```diff
@@ -16,4 +16,19 @@ if (isNil "support_specialized_training_available") then {
 
 if (isNil "support_armory_available") then {
     support_armory_available = false;
-};
\ No newline at end of file
+};
+
+/*
+    Support Timeout variables
+    Variables that hold the next VALID time to use that supports (in seconds)
+    Supports should set the next valid time as:
+    support_exampleSupport_timeout = *now* + xyz seconds;
+*/
+
+support_mortar_timeout = 0;
```

Comment:
optimally you would use a OFPEC tag for variables. But doesn't matter as much in mission scripts.